x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/XPackPlugin.java

```diff
@@ -403,9 +403,9 @@ public static Path resolveConfigFile(Environment env, String name) {
      * of SSLContexts when configuration files change on disk.
      */
     private SSLService createSSLService(Environment environment, ResourceWatcherService resourceWatcherService) {
-        final Map<String, SSLConfiguration> sslConfigurations = SSLService.getSSLConfigurations(environment.settings());
+        final Map<String, SslConfiguration> sslConfigurations = SSLService.getSSLConfigurations(environment);
```

Comment:
This capitalisation change is actually the core of the PR.
- `SSLConfiguration` is the old X-Pack version
- `SslConfiguration` is the new ssl-config version